openshift/operators/sso-operator.yml

```diff
@@ -2,20 +2,16 @@ apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
   name: rhsso-operator
-  namespace: eap
 spec:
   targetNamespaces:
-  - eap
-
+  - coolstore3
```

Comment:
This should be coolstore not coolstore3

Reply:
Also, I noticed the change from jdk 11 to jdk 8, what is the reason for this?

Reply:
fixed.

That's annoying, this is the only mention of the namespace that prevents to be able to deploy the app in any namespace. 
Unfortunately, RH-SSO operator does not support installation in all namespace and I've not been able to find a way to make the operator group configuration contextual to its namespace.